lambda-functions/Lambda/SQLAlchemy/models.py

```diff
@@ -0,0 +1,29 @@
+from sqlalchemy import create_engine, Column, Integer, JSON, String
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import sessionmaker
+from db_utils import db_host
+
+engine = create_engine(db_host)
+Base = declarative_base
+Session = sessionmaker(bind=engine)
+
+class Post(Base):
```

Comment:
The context of this class is a bit ambiguous, so a name like `RedditPost` might make this easier to understand a first glance. Consider renaming?